CODEOWNERS

```diff
@@ -9,18 +9,18 @@
 /src/WebSdk/ @vijayrkn
 /src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/ @vijayrkn
 
-/src/BlazorWasmSdk @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Assets/TestProjects/BlazorHosted @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Assets/TestProjects/BlazorHostedRID @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Assets/TestProjects/BlazorWasmMinimal @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Assets/TestProjects/BlazorWasmWithLibrary @captainsafia, @pranavkm, @mkArtakMSFT
-
-/src/RazorSdk @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Tests/Microsoft.NET.Sdk.Razor.Tests @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Tests/Microsoft.NET.Sdk.Razor.Tool.Tests @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Assets/TestPackages/PackageLibraryDirectDependency @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Assets/TestPackages/PackageLibraryTransitiveDependency @captainsafia, @pranavkm, @mkArtakMSFT
+/src/BlazorWasmSdk/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Assets/TestProjects/BlazorHosted/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Assets/TestProjects/BlazorHostedRID/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Assets/TestProjects/BlazorWasmMinimal/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Assets/TestProjects/BlazorWasmWithLibrary/ @captainsafia, @pranavkm, @mkArtakMSFT
+
+/src/RazorSdk/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Tests/Microsoft.NET.Sdk.Razor.Tests/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Tests/Microsoft.NET.Sdk.Razor.Tool.Tests/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Assets/TestPackages/PackageLibraryDirectDependency/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Assets/TestPackages/PackageLibraryTransitiveDependency/ @captainsafia, @pranavkm, @mkArtakMSFT
 /src/src/Assets/TestProjects/Razor* @captainsafia, @pranavkm, @mkArtakMSFT
```

Comment:
Do we need a trailing slash on this path?

Reply:
```suggestion
/src/src/Assets/TestProjects/Razor*/ @captainsafia, @pranavkm, @mkArtakMSFT
```

Reply:
Just checked and these are indeed folders so we do need that. Should work.